### ISSUE TITLE
Packager copies empty folders (from node_modules and ignored patterns) 

### ISSUE THREAD

**Issue by mahnunchik** [electron/electron-packager#1159]
### Preflight Checklist

* [x] I have read the [contribution documentation](https://github.com/electron/electron-packager/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project follows, as appropriate.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

### Issue Details

* **Electron Packager Version:**
  *  14.2.1
* **Electron Version:** v8.3.1
  * 
* **Operating System:**
  *  any

### Expected Behavior

Packager doesn't copy empty folders.

### Actual Behavior

Packager copies empty folders:

* Ignored devDependancies modules
* folders ignored by `ignore` option

### To Reproduce

* Install some `devDependancies` and package app.
* Ignore whole folder `/fortest/`

### Additional Information

Content of folders was not copied.

**Comment by mahnunchik** [comment]
```js
module.exports = {
  packagerConfig: {
    ignore: [
      '/resources/',
    ],
  }
}
```

For example the config above produces at least `resources`, `node_modules/@electron`, and `node_modules/@electron-forge` in the output folder or packaged asar.

**Comment by mahnunchik** [comment]
Ignoring of `resources` folder may be solved by specifying `'/resources'` (without leading slash).

So, I think that `ignore` behaviour should be more predictable or/and `include` option should be invented: https://github.com/electron/electron-packager/issues/972

**Comment by malept** [comment]
This would likely be fixed by #972.
